Blameless-Postmortems.md

```diff
@@ -0,0 +1,19 @@
+Blameless postmortems should be used to discuss issues that have affected our customers and how we can help to avoid similar situations from occurring in the future. The Postmortem should include all involved at their earliest opportunity after the event. 
+
+Please use this format for all Blameless Postmortems so that we have a consistent set of data on how we handle situations. Please add all notes from Blameless Postmortem to the Jira project (https://holidayextras.jira.com/secure/RapidBoard.jspa?projectKey=IR&rapidView=332&view=planning.nodetail), for reference.
```

Comment:
Would agree with this as it makes sense that SB post-mortems still get logged on our Jira system.